libs/input-mapping/src/Helper/LoadTypeDecider.php

```diff
@@ -20,4 +20,18 @@ public static function canClone(array $tableInfo, string $workspaceType, array $
         }
         return true;
     }
+
+    public static function canUseView(array $tableInfo, string $workspaceType, array $exportOptions): bool
+    {
+        if ($tableInfo['isAlias']) {
+            return false;
+        }
+
+        if (($tableInfo['bucket']['backend'] !== $workspaceType) ||
```

Comment:
Co plz znamena tahleta podminka? To je pro pripad, ze workspace ma jiny typ nez Storage backend?
Jakoze loadujes ze Snowflake do Redshiftu nebo tak neco? :)